src/Visual_TEST.cc

```diff
@@ -285,3 +285,38 @@ TEST(DOMVisual, SetLaserRetro)
   EXPECT_TRUE(visual.HasLaserRetro());
   EXPECT_DOUBLE_EQ(150, visual.LaserRetro());
 }
+
+/////////////////////////////////////////////////
+TEST(DOMVisual, ToElement)
+{
+  sdf::Visual visual;
+  visual.SetName("my-visual");
+  visual.SetCastShadows(true);
+  visual.SetTransparency(0.2);
```

Comment:
```suggestion
  visual.SetTransparency(0.2f);
```
https://build.osrfoundation.org/job/sdformat-ci-pr_any-windows7-amd64/2782/msbuild/new/